versions/v1/template/server.ts

```diff
@@ -1,120 +1,160 @@
+
+  const customerSchema = makeExecutableSchema({
+    typeDefs,
+    resolvers: safeResolvers,
+  });
+
+  const remoteExecutableSchemas = await Promise.all(
+    gqlServers.map((gqlServer) => createRemoteExecutableSchema(gqlServer))
+  );
+
+  // Merge the two schemas
+  return mergeSchemas({
+    schemas: [...remoteExecutableSchemas, customerSchema],
+  });
 };
 
-const createApolloServer = async (config: CreateServerConfig, typeDefs?: string = typeDefs, resolvers?: ResolversMap = resolvers): Promise<ApolloServer> => {
+const createApolloServer = async (
+  config: CreateServerConfig,
+  typeDefs?: string = typeDefs,
+  resolvers?: ResolversMap = resolvers
+): Promise<ApolloServer> => {
   const schema = await createNewSchema(config.graphql, typeDefs, resolvers);
-  return new ApolloServer({schema, introspection: true});
-}
+  return new ApolloServer({ schema, introspection: true });
+};
 
 interface ApolloServerContext {
   db: Record<string, any>;
   helpers?: HelpersMap;
 }
 
-const createApolloServerContext = async (config: CreateServerConfig, helpers?: HelpersMap = helpers): Promise<ApolloServerContext> => {
+const createApolloServerContext = async (
+  config: CreateServerConfig,
+  helpers?: HelpersMap = helpers
+): Promise<ApolloServerContext> => {
   const databases: Record<string, any> = {};
   for (const db of config.databases) {
-      databases[db.name] = await createSatsumaKnex(db);
+    databases[db.name] = await createSatsumaKnex(db);
   }
 
   const helpersSafe = deepCloneVMFunction(helpers, globalContext);
 
   return {
     db: databases,
     helpers: helpersSafe,
-  }
-}
+  };
+};
 
-export const createExpressMiddleware = async (config: CreateServerConfig, typeDefs?: string = typeDefs, resolvers?: ResolversMap = resolvers, helpers?: HelpersMap = helpers): Promise<express.RequestHandler> => {
-    const apolloServer = await createApolloServer(config, typeDefs, resolvers);
-    const apolloServerContext = await createApolloServerContext(config, helpers);
+export const createExpressMiddleware = async (
+  config: CreateServerConfig,
+  typeDefs?: string = typeDefs,
+  resolvers?: ResolversMap = resolvers,
+  helpers?: HelpersMap = helpers
+): Promise<express.RequestHandler> => {
+  const apolloServer = await createApolloServer(config, typeDefs, resolvers);
+  const apolloServerContext = await createApolloServerContext(config, helpers);
 
-    await apolloServer.start();
-    return expressMiddleware(apolloServer, {
-      context: async ({req}) => apolloServerContext,
-    });
+  await apolloServer.start();
+  return expressMiddleware(apolloServer, {
+    context: async () => apolloServerContext,
+  });
 };
 
-export const startStandaloneServer = async (config: CreateServerConfig, typeDefs?: string = typeDefs, resolvers?: ResolversMap = resolvers, helpers?: HelpersMap = helpers): Promise<string> => {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const startStandaloneServer = async (
```

Comment:
Not sure why this isn't used but I disabled the linting.

Reply:
It's used by the CLI & Querier I believe. Just not used within this package

Reply:
I suspect it's a linter bug. I aliased the apollo import with the same name and the lint error went away. `import { startStandaloneServer as startStandaloneServerApollo } from "@apollo/server/standalone";`